recipes/aws-crt-cpp/all/conanfile.py

```diff
@@ -78,10 +81,13 @@ def package(self):
     def package_info(self):
         self.cpp_info.filenames["cmake_find_package"] = "aws-crt-cpp"
         self.cpp_info.filenames["cmake_find_package_multi"] = "aws-crt-cpp"
+        self.cpp_info.set_property("cmake_file_name", "aws-crt-cpp")
         self.cpp_info.names["cmake_find_package"] = "AWS"
         self.cpp_info.names["cmake_find_package_multi"] = "AWS"
+        self.cpp_info.set_property("cmake_target_name", "AWS")
```

Comment:
no set_property() for the moment please, or use absolute target (conan >= 1.43.0).

Reply:
OK, I will revert.

Reply:
Fixed